04-deny-traffic-from-other-namespaces.md

```diff
@@ -62,24 +62,23 @@ Query this web service from the `default` namespace:
 
 ```sh
 $ kubectl run test-$RANDOM --namespace=default --rm -i -t --image=alpine -- sh
```

Comment:
you should've changed these pods to secondary, then?

Reply:
Correct